src/Core/Paginator.Core.csproj

```diff
@@ -1,28 +1,37 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <AssemblyName>Nudes.Paginator.Core</AssemblyName>
-    <RootNamespace>Nudes.Paginator.Core</RootNamespace>
-    <Authors>Patrick Ens</Authors>
-    <Company>Nudes</Company>
-    <PackageLicenseFile>LICENSE</PackageLicenseFile>
-    <PackageProjectUrl>https://github.com/nudes-senaisp/Paginator</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/nudes-senaisp/Paginator</RepositoryUrl>
-    <Description>Paginator is a lib to help with pagination in projects the Idea is to be used heavily on the backend but as well be used on clients like blazor and xamarin</Description>
-    <RepositoryType>git</RepositoryType>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-  </PropertyGroup>
+	<PropertyGroup>
+		<TargetFramework>netstandard2.1</TargetFramework>
+		<AssemblyName>Nudes.Paginator.Core</AssemblyName>
+		<RootNamespace>Nudes.Paginator.Core</RootNamespace>
+		<LangVersion>11</LangVersion>
+		<Authors>Patrick Ens</Authors>
+		<Company>Nudes</Company>
+		<PackageLicenseFile>LICENSE</PackageLicenseFile>
+		<PackageProjectUrl>https://github.com/nudes-senaisp/Paginator</PackageProjectUrl>
```

Comment:
We can change this uri

Reply:
We should probably cross check the other libraries to see if they have the wrong Url as well